frontend/packages/dev-console/integration-tests/features/build-config/edit-build-config.feature

```diff
@@ -0,0 +1,89 @@
+@odc-5008
+Feature: Form based edit for Build Configs
+              As a user, I can have form based edit for Build Configs to improve the user experience.
+
+        Background:
+            Given user is at developer perspective
+              And user has created or selected namespace "aut-form-edit-build-config"
+
+
+        @smoke @to-do
+        Scenario: Edit Build Config page: EBC-01-TC01
+            Given user has created a deployment workload "nodejs-ex-git1"
+             When user navigates to build tab
+              And user clicks on kebab menu for "nodejs-ex-git1" build config
+              And user clicks on Edit BuildConfig
+             Then user will see the Git Repository url, Image configuration, Environment Variables
```

Comment:
The name field should also be shown.

```suggestion
             Then user will see the Name Git repository url, Image configuration, Environment Variables
```

Btw. Isn't it easier later to implement this when you check things seperate?

```suggestion
             Then user will see the "Name" field
              And user will see the "Git repo URL" field
              And user will see the "Images" section
              And user will see the "Environment Variables" section
```

This is really just a question as I don't implemented these page objects and actions later.